src/geocel/g4/GeantGeoTrackView.hh

```diff
@@ -85,9 +85,14 @@ class GeantGeoTrackView
     CELER_FORCEINLINE Real3 const& dir() const { return dir_; }
     //!@}
 
-    // Get the volume ID in the current cell.
-    CELER_FORCEINLINE VolumeId volume_id() const;
-    CELER_FORCEINLINE int volume_physid() const;
+    // Get the volume ID in the lowest level volume.
+    inline VolumeId volume_id() const;
```

Comment:
Ah, so the `CELER_FORCEINLINE` is most clear as a tag on the definition, not the declaration, since its use really depends on how small the implementation is. I'm working to make this more consistent in the code: the one-line (1-2 operation) functions should probably be forced inline, and larger functions might not need to be. This is most relevant for debug/reldeb builds since the full optimization mode should take care of that mostly.